docs/tutorial/01-moveto-raw.html

```diff
@@ -0,0 +1,114 @@
+<!--
+1. Move the `will-move` divs to an new container with a class of `eoc` at the end of the chapter
+2. Add a section title with the class "os-title" and the value "End of Chapter Collations"
+3. Change `will-move` divs to sections
+
+Expected results:
+<div class="chapter">
+  <h1 data-type="document-title">Chapter Title</h1>
+  <div data-type="page">
+    <h2>Page Title</h2>
+    <div class="will-not-move">
+      <h3>Title</h3>
+      content
+    </div>
+  </div>
+  <div class="eoc">
+    <h2 class="os-title">End Of Chapter Collations
+    <div class="will-move">
+      <h3>Title</h3>
+      content
+    </div>
+  </div>
+</div>
+-->
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Move-to, creating containers</title>
+  </head>
+  <body>
+    <div data-type="chapter">
+      <h1 data-type="document-title">Chapter Title</h1>
+      <div data-type="page">
+        <h2 data-type="document-title">Chapter Title</h2>
+        <div class="will-not-move">
+          <h3>I didn't go anywhere</h3>
+          <p>
+            Content will not move <br />
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
```

Comment:
Does the lorem ipsum and extra elements help with the tutorial? I find the additional text distracting because I don't know if I should be paying attention to it or just ignoring it. Also, the HTML file is then more difficult to read because it is less information-dense.

Also, if these are removed then the "Expected Results" more closely match

Reply:
In my opinion, it more closely resembles a normal document with a lot of data to parse. To some extend, making the HTML harder to read is part of the training :) I also removed the expected results in favor of the baked html in an effort to avoid confusion